test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -360,6 +360,13 @@ Describe "Get-Help should find pattern alias" -Tags "CI" {
         $help.Category | Should -BeExactly "Alias"
         $help.Synopsis | Should -BeExactly "Where-Object"
     }
+
+    It "Help alias should be same as Get-Help alias" {
+        Set-Alias -Name testAlias1 -Value Where-Object
```

Comment:
Please remove the temp alias after the test.

Reply:
Existing line 344 removes the test alias

Reply:
Looking at the code, I'll move the creation of the alias to a BeforeAll statement